.gitignore

```diff
@@ -1,6 +1,6 @@
 # Build files
 .gradle/
-gradle.properties
```

Comment:
I don't think this ignore rule should be changed? The CI should run without committing the `gradle.properties` file.

Reply:
> I don't think this ignore rule should be changed? The CI should run without committing the `gradle.properties` file.

reverted back the change in this file.